### ISSUE TITLE
Bank Chest

### ISSUE THREAD

**Issue by DeltaEvo** [aresrpg/aresrpg#50]
- Open an double chest inventory when clicking on a ender chest
- Add multiple pages

**Comment by Sceat** [comment]
How would Banks works in a decentralized environment ? like in real life we hold peoples money on a dev wallet ? :grimacing:

**Comment by Sceat** [comment]
> How would Banks works in a decentralized environment ? like in real life we hold peoples money on a dev wallet ?

actually a NFT can't be stored in the player's bank, the only way to transfert to another account would be via a transaction, and in case the characters are also a token, well the player can also simply send his character to a wallet containing the stuff

**Comment by Sceat** [comment]
might be duplicate of #526

**Comment by Sceat** [comment]
This issue is basically about implementing a vanilla inventory with some specificactions:

- For now there will be only one bank chest limited to a vanilla double chest space #526 
- In the future there will be multiple fixed chests, so you can already store theses in the state as
```js
soul: 100,
// use your own bank functions, this is just an example
bank: Array.from({
	length: 1,
	0: Array.from({ length: 54 }) // the first bank chest content
})
```
- Handle vanilla in-chest gestures, stacks & movements (also be proactive and add those to the player inventory)
- Use #110 as an example on how to store items data (PR will come soon)

**Comment by Sceat** [comment]
closed in favor of #526
